Automattic/wp-calypso
# Update a8c deps

#### Changes proposed in this Pull Request
I released new versions of gridicons, social-logos, and `@automattic/react-virtualized` today with React 17 added to peer deps. This introduces those changes to Calypso. needed for #54793

#### Testing instructions
1. Run calypso locally (or in calypso.live)
2. visit `/devdocs/design/social-logo` and `/devdocs/design/gridicons` on the develop version of calypso. Compare those pages against the same pages on wpcalypso.wordpress.com and verify:
    - There should be no differences in the gridicons 
    - The facebook and medium icons should be updated in social logos
    - No other differences.
3. Test `read/search` (which uses react virtualized). Try loading results and using infinite scrolling. There shouldn't be any difference compared to trunk.
